packages/aws-cdk-lib/aws-sns/lib/topic.ts

```diff
@@ -236,6 +248,15 @@ export class Topic extends TopicBase {
       throw new Error(`signatureVersion must be "1" or "2", received: "${props.signatureVersion}"`);
     }
 
+    if (
+      props.tracingConfig &&
+      !Token.isUnresolved(props.tracingConfig) &&
+      props.tracingConfig !== 'PassThrough' &&
+      props.tracingConfig !== 'Active'
+    ) {
+      throw new Error(`tracingConfig must be "PassThrough" or "Active", received: "${props.tracingConfig}".`);
+    }
```

Comment:
The enumeration also allows your to remove this check

Reply:
removed.